Makefile

```diff
@@ -117,13 +125,21 @@ test-fmt:
 	git diff --exit-code || (git --no-pager diff && git checkout . && exit 1)
 .PHONY: test-fmt
 
-lint: get-libs
+lint: get-libs $(GOLINES)
 	./bin/golangci-lint run --config .golangci.yml
+	@output="$$($(GOLINES) -m 120 --dry-run --base-formatter=gofmt --ignore-generated .)"; \
```

Comment:
It would be cool if we could use https://github.com/mvdan/gofumpt here instead of `gofmt` since `golangci-lint` also verifies that the code has been formatted with this.

Reply:
I will see if that's possible. Out of curiosity: why is gofumpt better?

Reply:
it is stricter than `gofmt`. In a sense that code that passes `gofumpt` will also pass `gofmt` but not necessarily the other way round.